src/StolenObject.cpp

```diff
@@ -1,4 +1,7 @@
 #include "../include/StolenObject.h"
+#include <stdio.h>
```

Comment:
don't commit this; no standard io is going in the final project

Reply:
Temporary solution for debugging. Won't commit it in final product, but it is helpful in debugging.